.gitignore

```diff
@@ -2,12 +2,14 @@
 */target/.*
 */target/
 backend/dist
-backend/dist
-backend/dist
 node_modules
 cert.pem
 key.pem
 backend/cert.pem
 backend/key.pem
 .env
 .vscode
+backend/game.txt
+engine/base_good.tar.gz
+engine/base_good/
+engine/puzzle.txt
```

Comment:
might be nicer to put all of these in some directory like `fake_database` and just gitignore that, rather than individually gitignoring them here